### DISCUSSION TITLE
essay/2024/09/13/hall-of-fame

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#234] [Comments]
# essay/2024/09/13/hall-of-fame

조건문에 ! 하나를 빼먹어서 생긴 버그. 이 버그를 2년 동안이나 모르고 있다가 얼마 전에 발견했습니다.

https://jeho.page/essay/2024/09/13/hall-of-fame.html

**Comment by yuhaeun-la** [comment]
니가 만든 어플 편하더라~ 덕분에 도움이 되었다~ 라는 소리만 들어도 너무너무 기쁜데, 유저 인생의 큰 이벤트를 앱 서비스로 이루어내신건 얼마나 뿌듯하실지요. 아주 기분좋은 원동력이 될 것 같아요. 기쁜 소식이 비정기적으로 메일로 온다니 하하하. 재호님의 어느날은 종종 기쁨으로 가득차겠네요. 유저와 개발자간의 완벽한 선순환 ~ 버그 찾아내신것은 럭키비키에요🍀🍀

**Reply by BenjaminKim** [reply]
오, 안녕하세요. 잘 지내고 계시나요?
말씀하신대로 한 번씩 받는 메일에 기쁨으로 가득 찹니다. 정말 기분 좋습니다.
@yuhaeun-la 님께서도 어서 서비스 만드셔서 보여주세요~ 커피챗 기다리고 있습니다.
